README.md

```
@@ -308,6 +308,147 @@ const toolResult = await tool.invoke(
 - **Automatic fallbacks**: If you don't provide custom names, the tracer uses sensible defaults based on the LangChain component names
 - **Session linking**: Use `sessionId` to group multiple traces under the same user session for better analytics
 
+### AI SDK
+
+Use the built-in `wrapMaximAISDKModel` to wrap provider models and integrate Maxim observability and logging with your agents using Vercel AI SDK.
+
+```ts
+const model = wrapMaximAISDKModel(anthropic('claude-3-5-sonnet-20241022'), logger);
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)